apps/ts-minbar-test-react-components/package.json

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29281&commitId=4762d1afe737ac18e97c8ce918bce1bd411d5ec2&env=prod&clientType=FLUENTUI)


  <details><summary><b>TextField</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextField.Icon - RTL.default.chromium.png|62| Changed|
|TextField.Icon.default.chromium.png|70| Changed|
|TextField.Icon.hover.chromium.png|70| Changed|
|TextField.Icon - RTL.click.chromium.png|62| Changed|
|TextField.Icon.click.chromium.png|70| Changed|
|TextField.Icon - RTL.hover.chromium.png|62| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>